### PR TITLE
docs: add username to changeable attributes (DSP-1895)

### PR DESCRIPTION
**Pull request by irinaschubert**
resolves DSP-1895

### CODE REVIEW

File: docs/03-apis/api-admin/users.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ specified by the `id` in the request body as below:
 ### Update basic user information**
 
   - Required permission: SystemAdmin / self
-  - Changeable information: email, given name, family name,
+  - Changeable information: username, email, given name, family name,
```
Comment (irinaschubert) [inline docs/03-apis/api-admin/users.md:114]: This is the only change I've done in this file. The rest is reformatting.

File: webapi/src/main/scala/org/knora/webapi/messages/admin/responder/usersmessages/UsersMessagesADM.scala [hunk 2]
```diff
@@ -77,29 +79,31 @@ case class CreateUserApiRequestADM(id: Option[IRI] = None,
...
-                                   email: Option[String] = None,
-                                   givenName: Option[String] = None,
-                                   familyName: Option[String] = None,
-                                   lang: Option[String] = None,
-                                   requesterPassword: Option[String] = None,
-                                   newPassword: Option[String] = None,
-                                   status: Option[Boolean] = None,
-                                   systemAdmin: Option[Boolean] = None) {
+ * Represents an API request payload that asks the Knora API server to update an existing user. Information that can
+ * be changed include the user's username, email, given name, family name, language, password, user status, and system admin
```
Comment (irinaschubert) [inline webapi/src/main/scala/org/knora/webapi/messages/admin/responder/usersmessages/UsersMessagesADM.scala:83]: This is the only change I've done in this file. The rest is reformatting.

File: webapi/src/main/scala/org/knora/webapi/messages/admin/responder/usersmessages/UsersMessagesADM.scala [hunk 3]
```diff
@@ -624,28 +645,26 @@ case class UserADM(id: IRI,
...
-    */
+   * Given an identifier, returns true if it is the same user, and false if not.
+   */
   def isSelf(identifier: UserIdentifierADM): Boolean = {
 
-    val iriEquals = identifier.toIriOption.contains(id)
-    val emailEquals = identifier.toEmailOption.contains(email)
+    val iriEquals      = identifier.toIriOption.contains(id)
+    val emailEquals    = identifier.toEmailOption.contains(email)
     val usernameEquals = identifier.toUsernameOption.contains(username)
```
Comment (mpro7) [inline webapi/src/main/scala/org/knora/webapi/messages/admin/responder/usersmessages/UsersMessagesADM.scala:656]: Although most of re-formatting done in the file makes sense, the one marked above I don't personally like and it's kind of the same "alignment idea" to arguments formatted here before. I opt to keep fields as it was before :)

Comment (irinaschubert) [reply webapi/src/main/scala/org/knora/webapi/messages/admin/responder/usersmessages/UsersMessagesADM.scala:656]: OK, I suggest that we do the reformatting in a separate PR and then reformat everything (we already tried that but then the tests didn't run through - so we need to be careful with this). Besides, I don't know how to re-do this now... (as it was done automatically by saving the file)

Comment (mpro7) [reply webapi/src/main/scala/org/knora/webapi/messages/admin/responder/usersmessages/UsersMessagesADM.scala:656]: It's matter of the formatter configuration. It should be rather easy to revert unwanted things.

**Review by BalduinLandolt — APPROVED** [review]

**Review by mpro7 — APPROVED** [review]
As we agreed, we need to agree on the formatting rules etc. until then we should avoid reformat the code, but in this single file we can have it as a reference, which will be adjusted soon.

**Pull request merged**